verilog/CST/verilog_matchers_test.cc

```diff
@@ -510,8 +510,6 @@ TEST(VerilogMatchers, AlwaysStatementHasEventControlStarTests) {
        EmbedInModule("always_comb begin a = b; end"), 0},
       {AlwaysStatementHasEventControlStar(),
        EmbedInModule("always @* begin a = b; end"), 1},
-      {AlwaysStatementHasEventControlStar(),
```

Comment:
Isn't `AlwaysStatementHasEventControlStar` technical still true ?

Reply:
`AlwaysStatementHasEventControlStar` is not true because it only searches for `*` without parentheses. Previously, it looked like `(*)` was simplified in the tree to just `*`. So this test is no longer true (I can restore it and set the expected value to `false`), instead I added a new test `AlwaysStatementHasParenthesis` that checks if `(*)` and its various variations parse into parentheses and `*` separately.

Reply:
Yes, with 'technical true' I essentially meant that the meaning of event-star is still the same even though it is a slightly different syntax.

Is this something where we need to adapt the [always_comb.cc](https://github.com/chipsalliance/verible/blob/master/verilog/analysis/checkers/always_comb_rule.cc) lint rule ?

Reply:
Ready to submit, after always_comb rule question is sorted.